.drone.yml

```diff
@@ -60,51 +85,98 @@ services:
   - name: dockersock
     path: /var/run
 
-volumes:
-- name: dockersock
-  temp: {}
+- name: snoop-rabbitmq
+  image: rabbitmq:3.7.3
 
----
-kind: pipeline
-name: publish to docker hub
+- name: snoop-tika
+  image: logicalspark/docker-tikaserver
 
-trigger:
-  status:
-  - success
-  event:
-  - push
-  - tag
+- name: search-es
+  image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+  # volumes:
+  # - name: search-es-v
+  #   path: /drone/src/volumes/search-es/data
+  environment:
+    discovery.type: single-node
 
-depends_on:
-- test
+- name: snoop-pg
+  image: postgres:12.2
+  environment:
+    POSTGRES_USER: snoop
+    POSTGRES_DATABASE: snoop
+    POSTGRES_PASSWORD: snoop
+  # volumes:
+  # - name: snoop-pg-v
```

Comment:
You can keep this temp volume here, since the service runs only once. I think it's better to have the data in a temp volume than on the final layer of the docker image, since that is overwritten in /opt as an overlayfs and the tmpfs will be a ramfs (much faster, no overhead).